packages/create-app/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@backstage/create-app",
+  "description": "Create app package for Backstage",
+  "version": "0.1.1",
```

Comment:
This should match the other versions in the monorepo, so something with an alpha tag :p